src/utils/parsing.js

```diff
@@ -27,7 +27,9 @@ export const getParams = (req) => {
   const contractName = params[0]
   const date = params[1].split('?')[0]
   const useEms = req.query.ems !== 'false'
-  return {contractName, date, useEms}
+  // logo is by default excluded in non-EMS contracts
+  const useLogo = useEms ? req.query.logo !== 'false' : req.query.logo === 'true'
```

Comment:
Can `req.query.logo` be just an empty string, `undefined` or both? If just one of them, this might be a little bit easier to read:
```suggestion
  const useLogo = req.query.logo !== '' ? req.query.logo === 'true' : useEms
```
If you don't know, let's keep it like this, since it is more foolproof :)

Reply:
I'd guess it can be both (undefined if the query parameter is not present, but could be set to '')